NEWS

```diff
@@ -89,11 +89,11 @@ Support for reading and writing private keys in PKCS#8 format.
 
 New API:
 `to_pem` and `to_der` now accept new parameter, `format`, to specify
-the format of the encoded files, either the dafault, legacy "ssleay", or
+the format of the encoded files, either the default, legacy "SSLeay", or
```

Comment:
`ssleay` is also the value of the parameter, and it is case-sensitive, so could you rather change the quote marks to backticks instead of the case?

Reply:
Done :-) :+1: